assets/constants.ts

```diff
@@ -5911,5 +5911,7 @@ export default {
   ],
 
   "DEV_MODE": true,
-  "EMULATOR_MODE": true
+  "EMULATOR_MODE": true,
+  "ASSET_TYPE_THUMBNAIL": "thumbnail",
```

Comment:
I don't think these are the only ones you'll need to move. See @anandwana001's PR.

Reply:
Yes, [PR 13159](https://github.com/oppia/oppia/pull/13159) is going to be merged first, I will resolve the conflicts after it has been merged. 
To make this PR work, lifting only this constant was sufficient.

Reply:
Yes, to make this PR successful, I have only 

> @srikanthkb I noticed that you haven't filled in the "Proof that changes are correct" section. Please explain, in that section, how you have tested your changes manually and verified that they're correct. Note that relying on the backend/CI tests is not sufficient.

Done.